### DISCUSSION TITLE
posts/add-giscus/

### DISCUSSION THREAD

**Discussion by giscus** [univdev/univdev.github.io#26] [Comments]
# posts/add-giscus/

개요 기술 블로그를 Github Pages로 이전하면서 정적인 웹사이트로 바뀐 이후, 기술 블로그에 방문하는 분들의 리액션을 알 수 있는 창구가 필요해졌습니다. 그래서 지난번에 게시물 하단에 applause-button를 추가했었죠.

https://univdev.page/posts/add-giscus/

**Comment by jsh6269** [comment]
:D

**Comment by YieumYoon** [comment]
giscus 사용하기 쉽네요! 감사합니다

**Comment by jameskim9509** [comment]
감사합니다!
